docker-app/qfieldcloud/core/admin.py

```diff
@@ -869,6 +876,10 @@ def export_applyjob_deltafile(
             },
         )
 
+    def rerun_job(self, request, object_id):
+        Job.objects.filter(pk=object_id).update(status="pending")
+        return redirect(request.META.get("HTTP_REFERER", "../.."))
```

Comment:
I guess the intention here is to return to the root of the admin? Does it really make sense? Why don't you stay on current job page, so you can see how it progress in statuses?

Also, I believe there is a better mechanism to move to another page than using the headers. Check the web UI views for examples.

Reply:
Oops, that was a leftover from a different approach I tried, `return redirect("..")` is the simplest and sufficient.

Reply:
`HttpResponseRedirect(".")` is something that was already used as a pattern in the code